.github/workflows/build.yaml

```diff
@@ -63,10 +63,9 @@ jobs:
 
       - name: Run tests
         shell: bash -l {0}
-        env:
-          DAF_BUTLER_PLUGINS: ${{ github.workspace }}/python/lsst/pipe/base/cli/resources.yaml
         run: |
           pytest -r a -v -n 3 --cov=lsst.pipe.base --cov=tests --cov-report=xml --cov-report=term --cov-branch
+          butler -h
```

Comment:
Should this be `butler register-instrument -h`?

Reply:
Ok. I was mostly using this to look at the output to make sure it is working but using one of the expected plugin names is better.